### ISSUE TITLE
Add basic information

### ISSUE THREAD

**Issue by MelanX** [Core447/PluginTemplate#1]
It would be good to add some basic knowledge how to use it, such as:

- [ ] Main requirements (`requirements.txt`)
- [ ] Required Python version
- [ ] Where to place it

At the moment, I'm trying to figure out which exact packages are required to write an own private plugin. Some information to set it up would be helpful. When this information was added, I already figured it out (I hope), but maybe it's good to know for other devs.

My idea would be just to add a README.md with this information.

**Comment by Core447** [comment]
Yes, I'll add more information to the readme. For now you can have a look the [docs](https://streamcontroller.github.io/docs/latest/).

**Comment by MelanX** [comment]
Good idea, thank you. I didn't see that.

**Comment by Core447** [comment]
Done with cb4f5050f103a4008f4b190501ee89127200d305
